.github/workflows/check-devbox.yaml

```diff
@@ -47,5 +47,5 @@ jobs:
         uses: jetpack-io/devbox-install-action@4a7f1d5332cc72057d5e8080edebfcdf652e642e # v0.8.0
         with:
           enable-cache: true
-          devbox-version: 0.9.0
+          devbox-version: 0.9.1
           sha256-sum: a4f66cacf6091530f3d51148df83a08353906496c8ada001b0edd7ac29226dc5
```

Comment:
Wait, changing the version but not the shasum works!?

What is this sha supposed to do in the case?
![image](https://github.com/gravitational/teleport/assets/16366487/039e1a02-a07e-4a02-a5a6-2c6f77962896)

Reply:
The reason this pin didn't work is that the input parameter should be `sha256-checksum` not `sha256-sum`: https://github.com/jetpack-io/devbox-install-action?tab=readme-ov-file#action-inputs - so we are not checking the checksum. Whoops.

But the docs for this action has an example that also uses `sha256-sum` - double whoops!

Reply:
Thanks, I updated both the checksum and the parameter name. 
I took the checksum from https://github.com/jetpack-io/devbox-install-action/pull/37/files.